presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraType.java

```diff
@@ -21,20 +25,48 @@
 
 import java.io.IOException;
 
+import static io.prestosql.plugin.cassandra.CassandraType.containsUnsupportedType;
+import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertTrue;
 
 public class TestCassandraType
 {
+    @Test
+    public void testContainsUnsupportedType()
```

Comment:
I wouldn't test `io.prestosql.plugin.cassandra.CassandraType#containsUnsupportedType`, for 2 reasons

- as we add more supported types, we will need to update the test
- the method is not that complex in itself (simple recursive visit over the type term), it's more about the assumption this method makes, how it interacts with other parts of the code base.

this is my opinion, though. Let me know if you disagree.

Reply:
Agreed. Let me remove it.